src/sentry/api/endpoints/project_filter_details.py

```diff
@@ -108,4 +108,4 @@ def put(self, request: Request, project, filter_id) -> Response:
             data={"state": returned_state},
         )
 
-        return Response(status=201)
+        return Response(status=200)
```

Comment:
Changed the response type for this PUT b/c it should only return 201 if new resource has been created

Reply:
I think this should be a `204: No content` ([source](https://melkornemesis.medium.com/http-status-code-decoded-a-guide-to-understanding-200-201-202-and-204-c08904e7f0fe))